src/main/resources/application.properties

```diff
@@ -1,7 +1,7 @@
 spring.datasource.url=jdbc:postgresql://localhost:5432/passbook
 spring.datasource.username=postgres
 spring.datasource.password=qwerty1201
-#spring.jpa.hibernate.ddl-auto=create-drop
+#spring.jpa.hibernate.ddl-auto=create
```

Comment:
Why are you always committing this again and again.

Reply:
Updated, As perviously discussed need this to be closed